src/common/store/pdfFiles/actions.js

```diff
@@ -18,3 +22,18 @@ export const storeFile = file => async dispatch => {
     }
     dispatch(addPdfFile({ file, hasBackup }));
 };
+
+export const storeFileWithLink = file => async dispatch => {
```

Comment:
1. This method doesn't use dispatch. Just remove it: 
```
export const storeFileWithLink = file => async dispatch => {
```
replace with:
```
export const storeFileWithLink = async file => {
```

And call it like that in `uploadPdfFile` method:
```
await storeFileWithLink(getFileDescriptor(getState()));
```


2. Better rename this method to `saveFileToStorage`